x-pack/plugins/cases/public/components/user_action_tree/user_action_host_isolation_comment_event.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useCallback } from 'react';
+import * as i18n from './translations';
+import { LinkAnchor } from '../links';
+import { useKibana } from '../../common/lib/kibana';
+import { SECURITY_SOLUTION_APP_ID } from './constants';
+
+interface EndpointInfo {
+  endpointId: string;
+  hostname: string;
+}
+
+interface Props {
+  type: string;
+  endpoints: EndpointInfo[];
+}
+
+const HostIsolationCommentEventComponent: React.FC<Props> = ({ type, endpoints }) => {
+  const activityLogPath = `/administration/endpoints?selected_endpoint=${endpoints[0].endpointId}&show=activity_log`;
+  const { getUrlForApp, navigateToUrl } = useKibana().services.application;
```

Comment:
Same here, I think this is going to change in the mentioned pr above. Just ensure that this will still working with those changes

Reply:
yeah, same here. We cant do it because `security_solution` is not a dependency plugin to `cases`